misc/config/config.yaml.nydus.tmpl

```diff
@@ -46,6 +46,8 @@ converter:
       builder: nydus-image
       # specify nydus format version, possible values: `5`, `6` (EROFS-compatible), default is `5`
       rafs_version: 5
+      # specify nydus compression algorithm, `lz4block` by default
+      # rafs_compressor: zstd
```

Comment:
Why not default value?

Reply:
it depends on the actual scenarios, I'm not sure if it's optimal for internal network.